worker/src/lib/HeliconeProxyRequest/forwarder.ts

```diff
@@ -73,13 +75,33 @@ export async function proxyForwarder(
   }
 
   if (cacheSettings.shouldReadFromCache) {
```

Comment:
`organization_id` is only used within these two block.

This is kind of a nit, but defining top scoped variables that are mutable is a pretty bad anti pattern.

Instead I would always try to use `const` if possible. 

```typescript

if (cacheSettings.shouldReadFromCache) {
    const { data: auth, error: authError } = await request.auth();
    if (authError == null) {
      const db = new DBWrapper(env, auth);
      const { data: orgData, error: orgError } = await db.getAuthParams();
      
      
      if (orgError === null) {
        console.error("Error getting org", orgError);
        ctx.waitUntil(
            recordCacheHit(
              cachedResponse.headers,
              env,
              new ClickhouseClientWrapper(env),
              orgData.organizationId;
            )
        );
        return cachedResponse;
      } else {
        console.error("MY ERROR");
      }
    }
    
  }

Reply:
something like this. You can also move it to another function to make it easier if you want

Reply:
Since we are using it for 2 place I removed the bad anti-patern with this commit https://github.com/Helicone/helicone/pull/1174/commits/eae63d83ca5b50a8070bff828897e5120edbdd1d. 
Until we can get OrgID from `Request Wrapper` this seems like the best way.